test/e2e/components/vl-http-error-message.js

```diff
@@ -1,51 +1,58 @@
 const { VlElement } = require('vl-ui-core');
 const { VlTypography } = require('vl-ui-typography');
+const { VlButton } = require('vl-ui-button');
 const { By } = require('selenium-webdriver');
 
 class VlHttpErrorMessage extends VlElement {
-    async getImageSource() {
-        const smallImage = await this.shadowRoot.findElement(By.css('#image-small'));
-        const normalImage = await this.shadowRoot.findElement(By.css('#image-normal'));
-        const smallImageSrc = await smallImage.getAttribute("src");
-        const normalImageSrc = await normalImage.getAttribute("src");
-        if (smallImageSrc != normalImageSrc) {
-        	throw new Error("Small en normal image src moeten gelijk zijn");
-        }
-        return smallImageSrc;
+    async getTitle() {
+        return (await this._getTitle()).getText();
     }
 
-    async getImageAlt() {
-    	const smallImage = await this.shadowRoot.findElement(By.css('#image-small'));
-    	const normalImage = await this.shadowRoot.findElement(By.css('#image-normal'));
-    	const smallImageAlt = await smallImage.getAttribute("alt");
-    	const normalImageAlt = await normalImage.getAttribute("alt");
-        if (smallImageAlt != normalImageAlt) {
-        	throw new Error("Small en normal image src moeten gelijk zijn");
-        }
-    	return smallImageAlt;
+    async getContent() {
+        return (await this._getTypography()).getText();
```

Comment:
moet dat niet getInnerHTML zijn dan?

Reply:
Als afnemer ben je niet zozeer geïnteresseerd in de HTML van het bericht lijkt me? Als dat wel zo is, kan er een getInnerHTML functie voorzien worden.